packages/components/src/components/wallet/icon/icon.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classNames from 'classnames';
+import { WalletSmall } from '../small';
+import './icon.scss';
+
+type TWalletIconProps = {
+    account_icon?: string;
+    className?: string;
+    wallet_bg?: string;
+    wallet_icon?: string;
+    account_size?: string;
+    wallet_size?: string;
+};
+
+const WalletIcon = ({
+    wallet_icon,
+    account_icon,
+    wallet_bg,
+    className,
+    account_size = '1.6rem',
+    wallet_size = '1rem',
+}: TWalletIconProps) => {
+    const is_solo = !wallet_icon || !account_icon;
+    const icon = wallet_icon || account_icon || '';
+
+    if (!wallet_icon && !account_icon) {
+        return null;
+    }
+
+    return (
+        <div className={classNames('wallet-icon', { className, 'wallet-icon--merged': !is_solo })}>
+            {is_solo ? (
+                <WalletSmall bg={wallet_bg} icon={icon} />
+            ) : (
+                <React.Fragment>
+                    <WalletSmall
```

Comment:
One of them is App Card small variant, right?

Reply:
Wallet Small Component handles both.

Reply:
We'll be having separate components for AppCard and WalletCard. So, their small variants should also be handled separately.
I think it'll be better to just use `Icon` component here instead of WalletSmall.